app/models/saved_claim/disability_compensation.rb

```diff
@@ -30,8 +28,11 @@ def to_submission_data(user)
 
     form526_uploads = form526['form526'].delete('attachments')
 
+    # Subclass is expected to implement #translate_data
```

Comment:
you could enforce abstract methods being implemented by the subclass by raising an error within the super class 
``` ruby
def translate_data(user, form526); raise 'Subclass is expected to implement #translate_data'; end
```